src/utils/global/globalHooks.ts

```diff
@@ -0,0 +1,22 @@
+import type { Application } from '../../app/Application';
+import type { Renderer } from '../../rendering/renderers/types';
+
+interface Globals
+{
+    __PIXI_APP_INIT__?: (arg: Application | Renderer) => void;
+    __PIXI_RENDERER_INIT__?: (arg: Application | Renderer) => void;
+}
+
+export function pixiRendererCreated(renderer: Renderer)
+{
+    const gThis = globalThis as unknown as Globals;
+
+    gThis.__PIXI_RENDERER_INIT__?.(renderer);
+}
+
+export function pixiAppCreated(app: Application)
+{
+    const gThis = globalThis as unknown as Globals;
+
+    gThis.__PIXI_APP_INIT__?.(app);
+}
```

Comment:
Why not use `declare global` here instead to avoid casting?

```suggestion
declare global
{
    var __PIXI_APP_INIT__:undefined | ((arg: Application | Renderer) => void);
    var __PIXI_RENDERER_INIT__: undefined | ((arg: Application | Renderer) => void);
}

export function pixiRendererCreated(renderer: Renderer)
{
    globalThis.__PIXI_RENDERER_INIT__?.(renderer);
}

export function pixiAppCreated(app: Application)
{
    globalThis.__PIXI_APP_INIT__?.(app);
}
```

Also, not crazy about the names with the "pixi" prefix. Maybe `rendererCreatedHook` and `appCreatedHook`?